pkgs/development/haskell-modules/configuration-ghc-8.10.x.nix

```diff
@@ -139,4 +139,10 @@ self: super: {
   # Not possible to build in the main GHC 9.0 package set
   # https://github.com/awakesecurity/spectacle/issues/49
   spectacle = doDistribute (markUnbroken super.spectacle);
+
+  # fails on missing doccheck dependency on ghc < 9.0
+  autodocodec = dontCheck super.autodocodec;
+
+  # test dependency sydtest is broken
+  cabal2json = dontCheck super.cabal2json;
```

Comment:
Can you report the sydtest problem upstream? The author seems to be using nixpkgs so hopefully has some interest in fixing this problem.

Reply:
It's a version conflict problem, likely introduced by our 2nix strategy. Not sure if reporting that upstream makes any sense?
Here the error:
```
sydtest>     This package indirectly depends on multiple versions of the same package. This is very likely to cause a compile failure.
sydtest>       package safe-coloured-text-terminfo (safe-coloured-text-terminfo-0.0.0.0-2HeLg8kUnwOGIkU7KlhuP) requires safe-coloured-text-0.1.0.0-BKgZSUgXMWV6HniKZGXuey
sydtest>       package sydtest (sydtest-0.11.0.0) requires safe-coloured-text-0.2.0.1-6O7WFEWsxZ6Jojc9khvXGd
sydtest>       package sydtest (sydtest-0.11.0.0) requires safe-coloured-text-0.2.0.1-6O7WFEWsxZ6Jojc9khvXGd
sydtest>       package sydtest (sydtest-0.11.0.0) requires safe-coloured-text-0.2.0.1-6O7WFEWsxZ6Jojc9khvXGd
sydtest>       package autodocodec-yaml (autodocodec-yaml-0.2.0.0-2jM0YjpBKVy79LxilwhM3x) requires safe-coloured-text-0.2.0.1-6O7WFEWsxZ6Jojc9khvXGd
```

Reply:
Okay fine, maybe update the comment that this override will be obsolete starting with Stackage LTS 20.